cypress/support/commands.js

```diff
@@ -31,3 +31,20 @@ Cypress.on('uncaught:exception', (err, runnable) => {
     // failing the test
     return false
   })
+  Cypress.Commands.add('checkBrokenLinks', () => {
+    cy.get('a').each((link) => {
+      const url = link.prop('href');
+  
+      if (url && url !== '#' && !url.startsWith('javascript:')) {
+        cy.request({
+          url,
+          failOnStatusCode: false,
+        }).then((response) => {
+          if (response.status === 404) {
+            cy.log(`Broken Link: ${url}`);
```

Comment:
would it be better if you add assertion instead of log here? something like this: 
`if (response.status === 404) {
          cy.log(`Broken Link: ${url}`);
          expect(response.status).to.eq(404, `Broken Link: ${url}`);
        } else {
          expect(response.status).to.be.lessThan(400, `Invalid response status code for link: ${url}`);
        }`

wdyt?